plugins/qtosgrave/osgrenderitem.cpp

```diff
@@ -261,11 +355,13 @@ void KinBodyItem::Load()
         OSGGroupPtr posglinkroot = new osg::Group();
         posglinkroot->setName(str(boost::format("link%d")%porlink->GetIndex()));
 
-        OSGMatrixTransformPtr posglinktrans = new osg::MatrixTransform();
-        SetMatrixTransform(*posglinktrans, tbodyinv * porlink->GetTransform());
-        posglinktrans->setName(str(boost::format("link%dtrans")%porlink->GetIndex()));
```

Comment:
Dont forget to restore this code. We dont wanna touch this class here, we just want to add DrawLabel functionality